### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by omahs**
fix: typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Arbitrum SDK
 
-Typescript library for client-side interactions with Arbitrum. Arbitrum SDK provides common helper functionaliy as well access to the underlying smart contract interfaces.
+Typescript library for client-side interactions with Arbitrum. Arbitrum SDK provides common helper functionality as well as access to the underlying smart contract interfaces.
```
Comment (bartlomiej94) [inline README.md:3]: We can also change `Typescript` to `TypeScript`

File: README.md
[the remark this answers is not in the document]
Comment (omahs) [reply README.md:1]: Sure! I'm adding these, nice catch!

**Comment by omahs** [comment]
@cla-bot check

**Review by fionnachan — APPROVED** [review]
LGTM, thanks for your contribution!

**Comment by fionnachan** [comment]
By the way, @omahs have you signed the CLA?

**Comment by omahs** [comment]
Hey @fionnachan yup I signed it when I opened the PR

**Review by brtkx — APPROVED** [review]

**Pull request merged**